webviews/editorWebview/index.css

```diff
@@ -212,9 +212,11 @@ body .comment-container .review-comment-header {
 }
 
 .status-item,
-.form-actions {
+.form-actions,
+.ready-for-review-text-wrapper {
 	display: flex;
 	gap: 8px;
+	align-items: center;
```

Comment:
Also fixes icon on first row of status checks (see 'before' screenshot) is misaligned.